src/Common/tests/System/Diagnostics/DebuggerAttributes.cs

```diff
@@ -32,12 +34,37 @@ internal static void ValidateDebuggerTypeProxyProperties(Type type, Type[] gener
             // Create an instance of the proxy type, and make sure we can access all of the instance properties 
             // on the type without exception
             object proxyInstance = Activator.CreateInstance(proxyType, obj);
-            foreach (var pi in proxyInstance.GetType().GetTypeInfo().DeclaredProperties)
-            {
-                pi.GetValue(proxyInstance, null);
-            }
```

Comment:
After removing this, we lost some validation due to usage no longer actually invoking the accessors. 😦